sqlglot/helper.py

```diff
@@ -119,14 +213,14 @@ def open_file(file_name):
 
 
 @contextmanager
-def csv_reader(table):
+def csv_reader(table: Table) -> t.Any:
```

Comment:
returns a csv reader then?

Reply:
I did some investigation about the csv module, and I think it doesn't expose its internal reader/writer classes. That's why I used `t.Any`. For example, I think the problem is made clear in [this thread](https://stackoverflow.com/questions/51264355/how-to-type-annotate-object-returned-by-csv-writer).